backend/routers/memories.py

```diff
@@ -14,6 +14,21 @@
 
 router = APIRouter()
 
+@router.put("/v1/memories/{memory_id}/mark-created", response_model=Memory, tags=['memories'])
```

Comment:
The endpoint (by convention) should probably be `.patch` and `{memory_id}/events`